src/Types.ts

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+export interface SpeakerProps {
+	picture: string;
+	name: string;
+	role?: string;
+	location?: string;
+	mail?: string;
+	phone?: string;
+}
+
+export interface CategoryProps {
+	name: string;
+	icon: string;
+	size?: number;
+	style?: React.CSSProperties;
+}
+
+export interface IconTextProps {
+	text: string;
+	icon: string;
+	size?: number;
+	style?: React.CSSProperties;
+}
```

Comment:
Maybe split the types into different files ?

Reply:
I'e created a folder named Types and pu each type definition in separated files